packages/html-reporter/src/metadataView.tsx

```diff
@@ -95,7 +95,18 @@ const GitCommitInfoView: React.FC<{ info: GitCommitInfo }> = ({ info }) => {
       <div className='hbox m-2 mt-1'>
         <div className='mr-1'>{author}</div>
         <div title={longTimestamp}> on {shortTimestamp}</div>
-        {info['ci.link'] && <><span className='mx-2'>·</span><a href={info['ci.link']} target='_blank' rel='noopener noreferrer' title='CI/CD logs'>logs</a></>}
+        {info['ci.link'] && (
+          <>
+            <span className='mx-2'>·</span>
+            <a href={info['ci.link']} target='_blank' rel='noopener noreferrer' title='CI/CD logs'>logs</a>
```

Comment:
```suggestion
            <a href={info['ci.link']} target='_blank' rel='noopener noreferrer' title='CI/CD logs'>Logs</a>
```

Reply:
Fine by me. @agg23 you reviewed this earlier and had some feelings about `logs`. How do you feel about the casing?

Reply:
I also complained about it not being capitalized, and said it should revert to it's previous name, "CI/CD Logs". @dgozman said it looked ugly, and thus the lowercase stuck.

Reply:
I prefer capitalized, that makes us it three of us. Let's capitalise it, we can still revisit.